src/cli_options.rs

```diff
@@ -58,9 +58,10 @@ impl Command for Cbsh {
                 "name to show in the shell",
                 None,
             )
-            .switch(
+            .named(
                 "password",
-                "use to specify a password to use for authentication",
+                SyntaxShape::String,
+                "use to trigger a prompt to specify a password to use for authentication",
```

Comment:
I think with the change, this line isn't quite right, is it?  It's provide the password or prompt it `-` is specified.